hazelcast-jet-hadoop/src/main/java/com/hazelcast/jet/connector/hadoop/WriteHdfsP.java

```diff
@@ -89,18 +94,38 @@ public boolean isCooperative() {
      * @return {@link ProcessorMetaSupplier} supplier
      */
     public static ProcessorMetaSupplier writeHdfs(JobConf jobConf) {
-        return new MetaSupplier(jobConf);
+        return new MetaSupplier(jobConf, identity(), identity());
     }
 
+    /**
+     * Returns a meta-supplier of processors that write HDFS files.
+     *
+     * @param jobConf     JobConf for writing files with the appropriate output format and path
+     * @param keyMapper   mapper which can be used to map the key to another value
+     * @param valueMapper mapper which can be used to map the value to another value
+     * @return {@link ProcessorMetaSupplier} supplier
+     */
+    public static <T, R, K, V> ProcessorMetaSupplier writeHdfs(JobConf jobConf,
```

Comment:
Javadoc should also include comments about `Writable` requirements in certain cases.